nuitka/build/static_src/CompiledFunctionType.c

```diff
@@ -570,7 +602,12 @@ PyTypeObject Nuitka_Function_Type = {
 #endif
 };
 
-void _initCompiledFunctionType(void) { PyType_Ready(&Nuitka_Function_Type); }
+void _initCompiledFunctionType(void) {
+    // PyModule_AddObject(Nuitka_Function_Type.m_module, "Meta", (PyObject *) &MetaType);
```

Comment:
Like I said, there is no module for them. We could consider having one, but right now, I see no point in that. Maybe it could allow sharing implementations across Nuitka of same version with multiple extension modules once we have ABI compatibility over time. But that is more a comment to myself.